src/PaymentGateways/Stripe/Repositories/AccountDetail.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Give\PaymentGateways\Stripe\Repositories;
+
+/**
+ * Class AccountDetail
+ *
+ * @package Give\PaymentGateways\Stripe\Repository
+ * @unreleased
+ */
+class AccountDetail {
+	/**
+	 * Return Stripe account id for donation form.
+	 *
+	 * @unreleased
+	 * @param int $formId
+	 *
+	 * @return \Give\PaymentGateways\Stripe\Models\AccountDetail
+	 */
+	public function getDonationFormStripeAccountId( $formId ) {
+		// Global Stripe account.
+		$accountId = give_get_option( '_give_stripe_default_account', '' );
```

Comment:
I would suggest moving the `$accontId` default closer to its use. For Example, it might fit better as an `else` to the `if` check. Otherwise it is easy to miss the intent if a form doesn't not have Stripe enabled.